src/oc_logger.hrl

```diff
@@ -10,3 +10,11 @@
 -define(LOG_INFO(Format, Args), error_logger:info_msg(Format, Args)).
 -define(LOG_ERROR(Format, Args), error_logger:error_msg(Format, Args)).
 -endif.
+
+-ifdef('21.0').
+-define(SET_LOG_METADATA(TraceId, SpanId),
+        logger:update_process_metadata(#{trace_id => io_lib:format("~32.16.0b", [TraceId]),
```

Comment:
mb add `_ =` for dialyzer? I'm not sure but sometimes I have to this

Reply:
I checked and it passed dialyzer on 21. Maybe it is smart enough to not care if the only possible return value from a function is `ok`? I'm going to ask around later since I have been confused about consistency of that dialyzer warning.

Reply:
cool, please ask, because _ = sometimes drives me crazy